docs/CHANGELOG.md

```diff
@@ -18,6 +18,17 @@ Test fixtures for use by clients are available for each release on the [Github r
   - tests/cancun/eip4844_blobs/test_blob_txs.py::test_blob_tx_attribute_calldata_opcodes
   - tests/cancun/eip4844_blobs/test_blob_txs.py::test_blob_tx_attribute_gasprice_opcode
   - tests/cancun/eip4844_blobs/test_blob_txs.py::test_blob_type_tx_pre_fork
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile_gas.py::test_point_evaluation_precompile_gas_usage
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_valid_precompile_calls
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_invalid_precompile_calls
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_point_evaluation_precompile_external_vectors
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_point_evaluation_precompile_calls
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_point_evaluation_precompile_gas_tx_to
+  - tests/cancun/eip4844_blobs/test_point_evaluation_precompile.py::test_point_evaluation_precompile_before_fork
+  - tests/cancun/eip7516_blobgasfee/test_blobgasfee_opcode.py::test_blobbasefee_before_fork
+  - tests/shanghai/eip3860_initcode/test_initcode.py::test_contract_creating_tx
+  - tests/shanghai/eip3860_initcode/test_initcode.py::TestContractCreationGasUsage
```

Comment:
I am being pedantic here, but should it not be:
```suggestion
  - tests/shanghai/eip3860_initcode/test_initcode.py::test_gas_usage
```

Reply:
This was because this was generated with a test class instead of a test function, so if we change the id here, it will point to an nonexistent test.

I think the case can be made that we change the ids of all of the test classes, just for them to be cohesive with the rest of the names, but I'm not really sure how to do it.